editor.planx.uk/src/pages/FlowEditor/components/Team/components/MembersTable.tsx

```diff
@@ -194,17 +194,19 @@ export const MembersTable = ({
               </StyledTableRow>
             ))}
             {showAddMemberButton && (
-              <TableRow>
-                <TableCell colSpan={5}>
-                  <AddButton
-                    onClick={() => {
-                      addUser();
-                    }}
-                  >
-                    Add a new editor
-                  </AddButton>
-                </TableCell>
-              </TableRow>
+              <Permission.IsPlatformAdmin>
+                <TableRow>
+                  <TableCell colSpan={5}>
+                    <AddButton
+                      onClick={() => {
+                        addUser();
+                      }}
+                    >
+                      Add a new editor
+                    </AddButton>
+                  </TableCell>
+                </TableRow>
+              </Permission.IsPlatformAdmin>
```

Comment:
We must have had a bad rebase somewhere because I swear I made this exact change a couple weeks ago ?? :eyes: 

Eg https://github.com/theopensystemslab/planx-new/pull/3713/files#diff-b5928b9d92361e26a33f6c0b9af0253b319eab4128cee43689cce0eb88cc87c5R151